nesta/core/luigihacks/batchgeocode.py

```diff
@@ -35,10 +36,10 @@ class GeocodeBatchTask(AutoBatchTask):
     test = luigi.BoolParameter()
     _routine_id = luigi.Parameter(default="DUMMY ROUTINE")
     db_config_env = luigi.Parameter()
-    city_col = luigi.Parameter()
-    country_col = luigi.Parameter()
+    city_col = SqlAlchemyParameter()
```

Comment:
gets rid of annoying warnings since it is assumed that `luigi.Parameter` is  `str`